installation/Makefile

```diff
@@ -5,7 +5,7 @@ API_POD = $$(kubectl get pod -l app=estela-django-api -o jsonpath="{.items[0].me
 API_IP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.spec.clusterIP}')
 LOCAL_API_IP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
 RESOURCES = db registry minio zookeeper kafka
-
+SERVICES ?= django-api celery-worker celery-beat redis build-project
```

Comment:
I think `estela-consumer` is missing here.

Reply:
It's already in https://github.com/bitmakerla/estela/blob/da6e95bf27b83c394b3be98f75131a428af1fff7/installation/Makefile#L36